sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/PageBlobAPITest.groovy

```diff
@@ -600,21 +603,20 @@ class PageBlobAPITest extends APISpec {
         thrown(StorageException)
     }
 
-    // TODO (alzimmer): Turn this on once paged responses become available
-    /*def "Get page ranges"() {
+    def "Get page ranges"() {
         setup:
         bu.uploadPages(new PageRange().start(0).end(PageBlobClient.PAGE_BYTES - 1),
             new ByteArrayInputStream(getRandomByteArray(PageBlobClient.PAGE_BYTES)))
 
         when:
-        Iterable<PageRange> response = bu.getPageRanges(new BlobRange(0, PageBlobClient.PAGE_BYTES))
+        Response<PageList> response = bu.getPageRanges(new BlobRange(0, PageBlobClient.PAGE_BYTES))
 
         then:
         response.statusCode() == 200
-        response.body().pageRange().size() == 1
-        validateBasicHeaders(headers)
-        headers.blobContentLength() == (long) PageBlobClient.PAGE_BYTES
-    }*/
+        response.value().pageRange().size() == 1
+        validateBasicHeaders(response.headers())
+        Integer.parseInt(response.headers().value("x-ms-blob-content-length")) == PageBlobClient.PAGE_BYTES
```

Comment:
Just curious, do we maintain the header value as "x-ms-blob-content-length" instead of "Content-Length"?

Reply:
This is a custom header that contains the blob size, Content-Length contains other information